docs/getting-started/macos-instructions.md

```diff
@@ -0,0 +1,88 @@
+# Getting Started with Spark .NET on MacOS
+
+These instructions will show you how to run a .NET for Apache Spark app using .NET Core on MacOSX.
+
+## Pre-requisites
+
+- Download and install the following: **[.NET Core 2.1 SDK](https://dotnet.microsoft.com/download/dotnet-core/2.1)** 
+- Download and install brew package manager using: 
+    ```shell
+    /usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+    ```
+- Download and install Java using 
+    ```shell
+    brew tap caskroom/versions
```

Comment:
when I run this I get the following:
`Error: caskroom/versions was moved. Tap homebrew/cask-versions instead.`

Should we just have a link to Java SDK and Spark binaries similar to other OS instructions so that we don't depend on `brew` behavior? (also, we don't have to worry about spark version installed by `brew install apache-spark`.)